<!-- Phidgets Gyroscope launch file -->

<launch>

  #### Nodelet manager ######################################################

  <node pkg="nodelet" type="nodelet" name="gyro_manager"
    args="manager" output="screen" />

  #### Gyroscope Driver #####################################################

  <node pkg="nodelet" type="nodelet" name="PhidgetsGyroscopeNodelet"
    args="load phidgets_gyroscope/PhidgetsGyroscopeNodelet gyro_manager"
    output="screen">

    # See README.md for more information on the parameters

    # optional param serial, default is -1
    <!-- <param name="serial" value="123456"/> -->

    # optional param hub_port, used if connected to a VINT hub
    <!-- <param name="hub_port" value="0"/> -->

    # optional param frame_id, default is "imu_link"
    <!-- <param name="frame_id" value="imu_link"/> -->

    # optional param angular_velocity_stdev, default is 0.095deg/s
    <!-- <param name="angular_velocity_stdev" value="0.001658" /> -->

    # supported data rates: 4 8 16 24 32 40 ... 1000 (in ms)
    <param name="data_interval_ms" value="4"/>

    # optional param publish_rate, defaults to 0
    <!-- <param name="publish_rate" value="0" /> -->

    # optional param server_name, required to locate remote network servers
    <!-- <param name="server_name" value="myPhidgetServer" /> -->

    # optional param server_ip, required to locate remote network servers
    <!-- <param name="server_ip" value="0.0.0.0" /> -->
  </node>

</launch>
